napari/resources/_icons.py

```diff
@@ -138,7 +139,7 @@ def write_colorized_svgs(
     colors: Iterable[Union[str, Tuple[str, str]]],
     opacities: Iterable[float] = (1.0,),
     theme_override: Optional[Dict[str, str]] = None,
-) -> Iterator[Tuple[str, str]]:
+):
```

Comment:
Is this change intended @Czaki?

Reply:
I made this change as this function returns nothing.